provider/aci.go

```diff
@@ -725,7 +725,7 @@ func (p *ACIProvider) CreatePod(ctx context.Context, pod *v1.Pod) error {
 		"Namespace":         pod.Namespace,
 		"UID":               podUID,
 		"CreationTimestamp": podCreationTimestamp,
-		"Priority":          pod.Annotations[priorityTypeAnnotation],
+		"virtual-kubelet.io-priority": pod.Annotations[priorityTypeAnnotation],
```

Comment:
Add this tag "virtual-kubelet.io-priority" only if priority is set

Reply:
using existing function `setContainerGroupPriority` to set priority tag only when the annotation is present, and setting the accepted values (`Regular` or `Spot`) in a case insensitive manner.